grpchealth_test.go

```diff
@@ -37,15 +37,16 @@ func TestHealth(t *testing.T) {
 	server := httptest.NewUnstartedServer(mux)
 	server.EnableHTTP2 = true
 	server.StartTLS()
-	defer server.Close()
+	t.Cleanup(server.Close)
```

Comment:
These tests work with `t.Parallel()` if we close the server in the cleanup method.